authentication/authentication_test.go

```diff
@@ -24,15 +24,19 @@ import (
 	"github.com/auth0/go-auth0/authentication/database"
 	"github.com/auth0/go-auth0/authentication/oauth"
 	"github.com/auth0/go-auth0/internal/client"
+	"github.com/auth0/go-auth0/management"
 )
 
 var (
 	domain                = os.Getenv("AUTH0_DOMAIN")
 	clientID              = os.Getenv("AUTH0_AUTH_CLIENT_ID")
 	clientSecret          = os.Getenv("AUTH0_AUTH_CLIENT_SECRET")
+	mgmtClientID          = os.Getenv("AUTH0_CLIENT_ID")
+	mgmtClientSecret      = os.Getenv("AUTH0_CLIENT_SECRET")
```

Comment:
I'd left these as matching what's used for the management tests to make it easier to reuse them (some folks might opt to set environment variables themselves instead of the .env file)